src/stories/containers/RecognizedDelegates/KeyStats/KeyDescriptionCards/ShadowDelegatesCard.tsx

```diff
@@ -0,0 +1,24 @@
+import styled from '@emotion/styled';
+import React from 'react';
+import GenericDelegateCard from '../../GenericDelegateCard';
+import { DescriptionDelegates, TotalDelegates } from './TotalRecognizedDelegatesCard';
+
+interface Props {
+  shadowTotal: string;
```

Comment:
this looks like a `number` to me, is there a reason to make it a string?

Reply:
done, you are right